forc-plugins/forc-client/src/util/pkg.rs

```diff
@@ -85,3 +102,366 @@ pub(crate) fn built_pkgs(path: &Path, build_opts: &BuildOpts) -> Result<Vec<Arc<
 
     Ok(built_pkgs)
 }
+
+pub fn generate_chunk_loader_contract_src(
+    abi: &ProgramABI,
+    num_chunks: usize,
+    chunk_contract_ids: &[String],
+) -> String {
+    let mut contract = String::new();
+    let types = create_type_map(abi);
+
+    // Contract header
+    contract.push_str("contract;\n\n");
+
+    // Configurables
+    contract.push_str("configurable {\n");
+    for i in 1..=num_chunks {
+        contract.push_str(&format!(
+            "    TARGET_{}: ContractId = ContractId::from({}),\n",
```

Comment:
We should have a limit on the number of chunks allowed. Otherwise, it's possible that the loader contract could exceed the allowable size of a contract.